core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -182,43 +184,68 @@ public boolean init(Endpoint endpoint) {
             }
 
             return true;
-        } catch (Exception e) {
+        } catch (Throwable t) {
```

Comment:
Question: Just want to know why you raised the level here. 😃

Reply:
Just wanted to make sure `failEarly()` is called at any cases.